### PR TITLE
fix(TBD-14446): fix NPE with dnd on hadoop metadata

### PR DESCRIPTION
**Pull request by AlixMetivier**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/model/hadoopcluster/util/EncryptionUtil.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 public class EncryptionUtil {
 	
 	public static String getValue(String value, boolean encrypt) {
-        if (!value.startsWith("context.") && value != null && value.length() > 0) {
+        if (value != null && !value.startsWith("context.") && value.length() > 0) {
```
Comment (dicarcab) [inline main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/model/hadoopcluster/util/EncryptionUtil.java:8]: does the class that calls this method risk to get the NPE on it's side ??. If the if is not done a null value is returned no ?

Comment (AlixMetivier) [reply main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/model/hadoopcluster/util/EncryptionUtil.java:8]: yes thats the expected behavior :)

Comment (dicarcab) [reply main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/model/hadoopcluster/util/EncryptionUtil.java:8]: well feels weird to expect a null value but the caller method can return null also so I suppose this is ok

**Review by dicarcab — APPROVED** [review]

File: main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/model/hadoopcluster/util/EncryptionUtil.java (hunk 1 above)
Comment (AlixMetivier) [reply main/plugins/org.talend.repository.hadoopcluster/src/org/talend/repository/model/hadoopcluster/util/EncryptionUtil.java:8]: well there was 2 ways of fixing this, in fact we added some new fields for dataproc, default value are not set for those fields so they are null
I could have set a default value but imho its best we handle null value, specially for the fact that dataproc has an impact on cloudera metadata ...
and to dissociate both is a bit longer than any of above possible fixes :D

**Pull request merged**